Updating the README to remove references to "global" key (#113)
fauna/vscode: README.md

Source code for the [Fauna VS Code extension](https://marketplace.visualstudio.com/items?itemName=fauna.fauna).

> This [extension](https://marketplace.visualstudio.com/items?itemName=fauna.fauna) allows users to view individual Fauna databases and run [FQL queries](https://docs.fauna.com/fauna/current/api/fql/) from directly inside VS Code.

Reviewer: ```suggestion
This [extension](https://marketplace.visualstudio.com/items?itemName=fauna.fauna) allows users to view individual Fauna databases and run [FQL queries](https://docs.fauna.com/fauna/current/api/fql/) directly inside VS Code.
```

![Extension settings](media/extension-settings.png)

> - `fauna.adminSecretKey`: The secret for a specific database

Reviewer: ```suggestion
- `fauna.adminSecretKey`: The secret for a specific database.
```

Add a period to the end of each of the preceding bullet points.

> - `fauna.domain`: The Fauna domain for the database's [region group](https://docs.fauna.com/fauna/current/learn/understanding/region_groups#how-to-use-region-groups) or `localhost` when using a local Fauna Dev Docker instance

Reviewer: ```suggestion
- `fauna.domain`: The Fauna domain for the database's [Region Group](https://docs.fauna.com/fauna/current/learn/understanding/region_groups#how-to-use-region-groups) or `localhost` when using a local Fauna Dev Docker instance.
```

> - `fauna.scheme`: One of `https` or `http` (only set to `http` when using a local Fauna Dev Docker instance)

Reviewer: ```suggestion
- `fauna.scheme`: One of `https` or `http`. Only set to `http` when using a local Fauna Dev Docker instance).
```

> - `fauna.port`: The port number for the Fauna endpoint (when using a Fauna Dev Docker instance, be sure to use the port passed to the `docker run` command - usually `8443`)

Reviewer: ```suggestion
- `fauna.port`: The port number for the Fauna endpoint. When using a Fauna Dev Docker instance, be sure to use the port passed to the `docker run` command - usually `8443`.
```

> - `fauna.graphqlHost`: The full URL for the Fauna GraphQL API (e.g. `https://graphql.fauna.com`)

Reviewer: ```suggestion
- `fauna.graphqlHost`: The full URL for the Fauna GraphQL API. The default is `https://graphql.fauna.com`. See [GraphQL Endpoints](https://docs.fauna.com/fauna/current/api/graphql/endpoints) for Region Group-specific URLs.
```

![Extension settings](media/extension-settings.png)

- `fauna.adminSecretKey`: The secret for a specific database.
- `fauna.domain`: The Fauna domain for the database's [Region Group](https://docs.fauna.com/fauna/current/learn/understanding/region_groups#how-to-use-region-groups) or `localhost` when using a local Fauna Dev Docker instance.
- `fauna.scheme`: One of `https` or `http`. Only set to `http` when using a local Fauna Dev Docker instance.

> - `fauna.port`: The port number for the Fauna endpoint. When using a Fauna Dev Docker instance, be sure to use the port passed to the `docker run` command - usually `8443`.

Reviewer: ```suggestion
- `fauna.port`: The port number for the Fauna endpoint. When using a Fauna Dev Docker instance, use the port passed to the `docker run` command - usually `8443`.
```

- `fauna.graphqlHost`: The full URL for the Fauna GraphQL API. The default is `https://graphql.fauna.com`. See [GraphQL Endpoints](https://docs.fauna.com/fauna/current/api/graphql/endpoints) for Region Group-specific URLs.

> WARNING: Be careful! To avoid exposing this secret, do not commit it to your local `.vscode` configuration.

### 2. Browse database

1. Click on the Fauna bird icon in the Activity bar on the far left. If you do not see the Activity Bar, select `View > Appearance > Show Activity Bar` from the VS Code menu.

> 2. You should now see a pane listing all of your database's contents, including child databases, indexes, collections, documents, and functions.

Reviewer: ```suggestion
2. You should now see a pane listing your database's contents, including child databases, indexes, collections, documents, and functions.
```
